json-rpc/src/data.rs

```diff
@@ -368,11 +367,9 @@ pub fn get_currencies(
 }
 
 /// Returns the number of peers this node is connected to
-pub fn get_network_status(role: &str) -> Result<u64, JsonRpcError> {
-    let peers = counters::DIEM_NETWORK_PEERS
-        .get_metric_with_label_values(&[role, "connected"])
-        .map_err(|e| JsonRpcError::internal_error(e.to_string()))?;
-    Ok(peers.get() as u64)
+pub fn get_network_status(_role: &str) -> Result<u64, JsonRpcError> {
+    // TODO: The underlying metric is deprecated, and we need a different way of communicating this number that doesn't need the peer Id
+    Ok(0)
```

Comment:
I know this is deprecated but could we instead use the new metrics?

Reply:
I didn't pull this out of draft for this reason.

It either needs a new metric, or we have to pull the peer Id from the network interface all the way through to here just to get the metric out.

Reply:
fine i'll stop looking till its not in draft :P

Reply:
Well, let's go forward with 0 for now, until a better replacement.  The metric currently doesn't give a valid number on VFNs, it'll flip flop between the two interfaces (VFN, Public).